acme/api/handler.go

```diff
@@ -306,6 +306,7 @@ func GetAuthorization(w http.ResponseWriter, r *http.Request) {
 	linker.LinkAuthorization(ctx, az)
 
 	w.Header().Set("Location", linker.GetLink(ctx, acme.AuthzLinkType, az.ID))
+	w.Header().Set("Cache-Control", "private, no-store")
```

Comment:
This is an ACME authorization object which includes a list of challenges with tokens. I don't believe the token is  sensitive since leaking it does not compromise the protocol. Is there another field on here of concern?

Reply:
I read into the protocol a bit and I see why caching the token wouldn't compromise the protocol. I'll remove the header here.